src/main/java/uk/gov/ch/model/officer/active/ActiveOfficerDetails.java

```diff
@@ -72,6 +72,26 @@
     private String residentialAddressRegion;
     @Column(name = "SECURE_INDICATOR")
     private String secureIndicator;
+    @Column(name = "corporate")
+    @JsonProperty("is_corporate")
+    private boolean corporate;
```

Comment:
is it possible for this value to be null in the db?
If it is, this might thrown a null pointer error.
Or it will default to false.

Might be better to use a Boolean instead so what is returned can more accurately reflect the state of the db

Reply:
i don't see any instances of null for that field in the db but can update this to use Boolean type just in case ?